### DISCUSSION TITLE
Confusion regarding "Disease frequency modifier" in World Settings.

### DISCUSSION THREAD

**Discussion by PaulBlay** [AtomicFox556/Cataclysm-EOD#152] [Q&A] [answered]
This setting is minimum 0, maximum 50 (I checked). 

The text description states (in part)...

A probability, in percents, to not catch an ambient disease such as a cold, flue or tetanus, any time it'd  otherwise happen. 0 means no 
reduction, 100 means ambient diseases never happen.

This seems to imply that at the maximum value (50) you catch ambient diseases half as often as you otherwise would. 

Is that so? I would have expected the maximum value to disable catching ambient diseases altogether.

**Comment by AtomicFox556** [comment, marked as answer]
That's a bug, yeah. Would be trivial to fix; just need to change 50 to 100 in options.cpp. I'll fix it once I can get some time.
